backend/tournesol/resources/export_readme.txt

```diff
@@ -187,4 +187,23 @@ List of columns:
   This field is computed by Tournesol's algorithms. It should represent the 
   uncertainty on the score value. For more detail, check out the algorithms
   implementation or the paper which describe them
-  https://arxiv.org/abs/2211.01179
\ No newline at end of file
+  https://arxiv.org/abs/2211.01179
+
+vouchers.csv
+------------
+
+This file contains all vouchers shared between users to obtain a higher trust score.
```

Comment:
To be explicit should we indicate that the trust is given by the sender to the receiver?

Reply:
Clarified in [62fb0b1](https://github.com/tournesol-app/tournesol/pull/1957/commits/62fb0b1160c108e53e09bb543d18b7ccc8ad5c8f)